src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Network/NewAzureReservedIP.cs

```diff
@@ -84,6 +85,14 @@ public string VirtualIPName
             set;
         }
 
+        [Parameter(Mandatory = false, Position = 5, ValueFromPipelineByPropertyName = true, ParameterSetName = ReserveInUseIPParamSet, HelpMessage = "IP Tags.")]
```

Comment:
@ankkum parameter `Slot` is already being used for position 5, so this will result in an error. We encourage a cmdlet have no more than four positional parameters for a cmdlet, so you will be fine just removing the position attribute from this new parameter.